notebooks/notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<p>There are multiple processes running in this chunk. Please add in line comments, i.e., to describe what conditional statements are meant to do in this particular case.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/repro-challenge-team-3/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>This code chunk has been merged into a script. We have also added some markdown text explaining what the entire script does. This was done because explaining things in many line comments was decreasing the notebook's readability.</p>

Reply:
closing conversation, resolved